app/test_main.py

```diff
@@ -1,3 +1,88 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "initial_cat_age,"
+        "initial_dog_age,"
+        "expected_human_age_cat,"
+        "expected_human_age_dog",
+        [
+            pytest.param(
+                0,
+                0,
+                0,
+                0,
+                id="test should return zero when age is zero"
+            ),
+            pytest.param(
+                14,
+                14,
+                0,
+                0,
+                id="test should return zero when age is less then 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                1,
+                1,
+                id="test should return one when age is equal "
+                   "or more then 15 and less then 24"
+            ),
+            pytest.param(
+                24,
+                24,
+                2,
+                2,
+                id="test should return two when age is equal "
+                   "or more then 24 and less then 28 for cat or 29 for dog"
+            ),
+            pytest.param(
+                28,
+                28,
+                3,
+                2,
+                id="test should return different values when age is "
+                   "more then 27 for cat and more then 28 for dog"
+            ),
+            pytest.param(
+                -4,
+                -3,
+                0,
+                0,
+                id="test should return zero when "
+                   "age is less then zero"
+            )
+        ]
+    )
```

Comment:
Add test cases with really large values and positive values for `initial_cat_age` and `initial_dog_age` that differ from one another according to the checklist.